topydo/cli/CLIApplicationBase.py

```diff
@@ -25,6 +25,7 @@
 from six.moves import input
 
 MAIN_OPTS = "ac:d:ht:v"
+NON_DESTRUCTIVE_COMMANDS = ('List', 'ListContext', 'ListProject')
```

Comment:
Suggestion is to rename to `READ_ONLY_COMMANDS`. I don't consider all other commands to be destructive, they just do what they're supposed to do ;)